psi4/src/psi4/libfock/PKmanagers.cc

```diff
@@ -1111,7 +1118,7 @@ void PKMgrYoshimine::compute_integrals(bool wK) {
             for (size_t j = 0; j <= i; ++j) {
                 int RR = sh_pairs[j].first;
                 int SS = sh_pairs[j].second;
-                if (tb[thread]->shell_significant(PP, QQ, RR, SS)) {
+                if (shell_significant(PP, QQ, RR, SS)) {
```

Comment:
so why does this no longer get applied per-thread? (read as naive, not insightful, question)

Reply:
The shell_significant function in PKMgrYoshimine is defined to do its shell significance test using the sieve_ variable in the base PKManager class instead of through the TwoBodyAOInt object. Theoretically, shell_significant in PKMgrYoshimine could be redefined to use tb[thread]'s shell_significant, though it would be somewhat clumsy since it would require passing { tb[thread] } as the optional argument for int.

Talking about it here though makes me wonder whether using sieve_ is the best approach for the shell_significant definition, however.

Reply:
The ERISieve class is deprecated in favor of having the `TwoBodyAOInt` object do the sieving. This was originally decided as part of the L2 discussion and has been reaffirmed on Slack just a week or two ago. I was in the process of getting rid of ERISieve when this PR came up.

We'll need to agree about the future of "what class is responsible for sieving" before this PR can be merged in.